corehq/apps/hqmedia/models.py

```diff
@@ -139,7 +141,12 @@ def attach_data(self, data, original_filename=None, username=None, attachment_id
                 # this should only be files that had attachments deleted while the bug
                 # was in effect, so hopefully we will stop seeing it after a few days
                 logging.error('someone is uploading a file that should have existed for multimedia %s' % self._id)
-            self.put_attachment(data, attachment_id, content_type=self.get_mime_type(data, filename=original_filename))
+            self.put_attachment(
+                data,
+                attachment_id,
+                content_type=self.get_mime_type(data, filename=original_filename),
+                domain=SHARED_DOMAIN,
```

Comment:
Just confirming that this is shared because the file_hash will be the same in case the same one is used for different domains.

Just one follow up question. In case we need to find blobs for a domain this wont be included then? like when we filter them in SQL, i guess that's how we would filter after this change?